src/mito2/src/test_util/scheduler_util.rs

```diff
@@ -65,7 +66,11 @@ impl SchedulerEnv {
     ) -> CompactionScheduler {
         let scheduler = self.get_scheduler();
 
-        CompactionScheduler::new(scheduler, request_sender)
+        CompactionScheduler::new(
+            scheduler,
+            request_sender,
+            Arc::new(CacheManager::new(0, 0, 0)),
```

Comment:
We'd better test the mito engine with the valid cache manager when it's ready.

Reply:
This util only mocks a scheduler for scheduler tests. The scheduler itself doesn't care about the cache manager so it's fine to provide an empty cache here.